test_settings.py

```diff
@@ -47,6 +47,7 @@ def root(*args):
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
+    'simple_history.middleware.HistoryRequestMiddleware',
```

Comment:
should this have a corresponding line in INSTALLED_APPS?  Not sure how testing succeeds if middleware is run without having the corresponding app installed.